tests/test-templates-export.php

```diff
@@ -85,7 +85,7 @@ public function test_chapter() {
 			$chapter_rendered
 		);
 		$this->assertContains( '<p class="chapter-number">1</p>', $chapter_rendered );
-		$this->assertContains( "<h1 class=\"chapter-title\">$title</h1>", $chapter_rendered );
+		$this->assertContains( "<h2 class=\"chapter-title\">$title</h2>", $chapter_rendered );
```

Comment:
@fdalcin this is a friendly commit to adapt the unit tests to the new changes since the tests were not present at the moment of this PR was created